src/open-uri.c

```diff
@@ -191,8 +191,16 @@ launch_application_with_uri (const char *choice_id,
 
   if (is_sandboxed (info))
     {
+      g_autoptr(GError) error = NULL;
+
       g_debug ("registering %s for %s", uri, choice_id);
-      ruri = register_document (uri, choice_id, FALSE, writable, NULL);
+      ruri = register_document (uri, choice_id, FALSE, writable, &error);
+      if (ruri == NULL)
+        {
+          g_warning ("Error registering %s for %s", uri, choice_id, error->message);
```

Comment:
This printf has two format placeholders, but takes 3 arguments. I think a `: %s` is missing at the end of the message.

Reply:
I guess this is what happens when you try to rush a PR just before going for the school pick up... Thanks, I'll fix that